SQL/tgstation_schema.sql

```diff
@@ -268,6 +283,7 @@ CREATE TABLE `player` (
   `computerid` varchar(32) NOT NULL,
   `lastadminrank` varchar(32) NOT NULL DEFAULT 'Player',
   `accountjoindate` DATE DEFAULT NULL,
+  `flags` int(11) NOT NULL,

```

Comment:
This should be an unsigned `SMALLINT(5)`; bitflags in byond are 16bit so their max value is 65535 just like `SMALLINT`